### ISSUE TITLE
RFC: Apply settings only to application windows?

### ISSUE THREAD

**Issue by listenlight** [TimurKiyivinski/gnome-shell-night-light-slider-extension#37]
Hi, I'm posting my first ever, and prob'ly half-baked, "RFC". I thought of this last night, where I have a nice color behind my minimal conky setup, that I'd like to keep the background color. I don't know how much trouble that would be, so I'd rather ask about it.

The browser and email clients, for instance, seems to benefit the most, having so much white background. But I like my Terminal apps as they are.

Basically I just want to say hi. Your extension is the first that made me dig deeger. I've got one setup, the barebones 'hello world', thinking about making my conky app an extension...
**night light slider full brightness (looks bloody else) **
![conky2](https://user-images.githubusercontent.com/12180/52940768-f300bc00-332c-11e9-8cc1-a5c1de7f9e3d.gif)

thank you.

**Comment by kiyui** [comment]
Hi, the extension makes use of the existing night light functionality in GNOME and as such, being able to select per-application dimming would not be possible.

This may be something that could be set up via a window compositor such as compton however, though I haven't tried achieving anything other than per-window opacity with it.

Also hi, and good luck publishing that extension! :smile:
